Mos/en.lproj/Main.strings

```diff
@@ -7,48 +7,48 @@
 */
 
 // Mointor
-"TUE-3v-CaJ.title" = "          Rendering this chart will affect the scrolling performance, keep close if not necessary.";
+"TUE-3v-CaJ.title" = "          Rendering this chart will affect scrolling performance. Close the window when not in use.";
 
 // PreferencesWindow
 // General
 "ITL-GD-CPd.title" = "General";
 "ILt-Gz-RiV.title" = "Scroll:";
 "weE-be-TAy.title" = "Smooth Scrolling";
-"G8a-97-8Ey.title" = "Let the mouse wheel scroll like a buttery.";
+"G8a-97-8Ey.title" = "Let the mouse wheel scroll smoothly like butter";
 "6FR-2r-rxl.title" = "Scroll Reverse";
-"VUb-ZM-hXo.title" = "Reverse the mouse wheel scroll direction.";
-"8sv-2I-VeW.title" = "Other Setting:";
+"VUb-ZM-hXo.title" = "Reverse the mouse wheel scroll direction";
+"8sv-2I-VeW.title" = "Miscellaneous:";
 "HZF-2o-Mrh.title" = "Launch on Login";
-"wY5-9W-iNv.title" = "Automaticly launch on MacOS login.";
+"wY5-9W-iNv.title" = "Automatically launch app on login";
 // Advanced
 "iW5-CD-5OC.title" = "Advanced";
-"NAp-3R-e8q.title" = "Shift Key:";
-"lzC-ub-pqJ.title" = "Transform the vertical scroll to horizontal";
+"NAp-3R-e8q.title" = "Toggle Key:";
+"lzC-ub-pqJ.title" = "Change vertical scrolling to horizontal scrolling";
 "akk-dX-tW4.title" = "Block Key:";
-"62p-76-okJ.title" = "Temporarily block the smoothing of the scroll";
+"62p-76-okJ.title" = "Temporarily block smooth scrolling";
 "bcp-Xx-rdX.title" = "Disable";
 "DXO-Xo-tJd.title" = "Disable";
 "YE2-UH-9Ku.title" = "Step:";
-"3Xk-SU-gqO.title" = "The minium scroll distance.";
+"3Xk-SU-gqO.title" = "Sets the minimum scroll distance";
```

Comment:
Use "set" not "sets" here.

Reply:
Either one works in this case. They are used evenly in everyday English.

Reply:
kidding me?

Reply:
This sentence is a simplified version of `This [the slider] sets the minimum scroll distance`. But one can remove the word `this` in the beginning because it is obvious as to what `this` is referring to (the slider). So we can render the sentence as `Sets the minimum scroll distance`. However `This set the minimum scroll distance` is grammatically incorrect.